docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -354,13 +354,16 @@ The `take over` mode can work correctly only if the source (taken from) inputs a
 `take_over.enabled: true` requires the `filestream` to have a unique ID.
 ::::
 
-
 This `take over` mode was created to enable smooth migration from
 deprecated `log` inputs to the new `filestream` inputs and to allow
 changing `filestream` input IDs without data re-ingestion.
 
 See [*Migrate `log` input configurations to `filestream`*](/reference/filebeat/migrate-to-filestream.md) for more details about the migration process.
 
+The previous configuration format `take_over: true`, while
+discouraged, is still supported to migrate state from the `log` input
+to `filestream`.
+
```

Comment:
Do we want to mention an ETA for when this compatibility layer will be removed? Otherwise, I expect people will simply stick with the older syntax.

Reply:
> Do we want to mention an ETA for when this compatibility layer will be removed?

Not at the moment, we haven't defined any deadline for this. I know internally we use this feature and the current "version filters" we have are based only on the major. So I can see this staying here until 10.x.